src/validateUrl.js

```diff
@@ -0,0 +1,32 @@
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const validateURL = (text, toCase) => {
+  const errors = [];
+  const correctExample = '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: '
```

Comment:
You can create an object with all your default error messages and use them as reference